DataPlotly/core/plot_settings.py

```diff
@@ -59,6 +59,7 @@ def __init__(self, plot_type: str = 'scatter', properties: dict = None, layout:
             'bins': 0,
             'selected_features_only': False,
             'visible_features_only': False,
+            'feature_subset_query': QgsProperty().toVariant(),
```

Comment:
What about simplifying to just:

```suggestion
            'filter': QgsProperty().toVariant(),
```